bash.exe.stackdump

```diff
@@ -0,0 +1,9 @@
+Stack trace:
+Frame        Function    Args
```

Comment:
No, not resolved. It's still in the code base. Look here https://github.com/bahdcoder/hello-books-deferral/tree/ft-view-books-page-166071085

Reply:
I just checked it now....it's not there